tests/src/DB.res

```diff
@@ -0,0 +1,50 @@
+open Belt
```

Comment:
`Core` is in there btw.

Reply:
Never used it before, should learn it first :joy: